### PR TITLE
Fuku02/fix

### PR DESCRIPTION
**Pull request by fuku02**
Thank you for your code review.
Fix pull requests.

1.ERROR: The named parameter 'maxLengthEnforced' isn't defined.
https://pub.dev/packages/input_history_text_field/score
MaxLengthEnforced => MaxLengthEnforcement

2.ERROR 'package:flutter/src/rendering/object.dart': Failed assertion: line 2462 pos 12: 'attached': is not true.
https://github.com/trashfeed/input_history_text_field/issues/5
Add OverlayEntry.remove()

3.Fix filterHistory
It seems to include the current input text in filterHistory.

### CODE REVIEW

File: example/ios/Runner/Info.plist [hunk 1]
```diff
@@ -41,5 +41,7 @@
 	</array>
 	<key>UIViewControllerBasedStatusBarAppearance</key>
 	<false/>
+	<key>CADisableMinimumFrameDurationOnPhone</key>
+	<true/>
```
Comment (trashfeed) [inline example/ios/Runner/Info.plist:45]: What was this added for? I need a supplemental explanation!

File: lib/src/widget/input_history_text_field.dart [hunk 2]
```diff
@@ -79,6 +79,12 @@ class InputHistoryTextField extends StatefulWidget {
   // enabled/disabled of input history
   final bool enableHistory;
 
+  /// show history list
+  final bool showHistoryList;
+
+  // enabled/disabled of filter history
+  final bool enableFilterHistory;
+
```
Comment (trashfeed) [inline lib/src/widget/input_history_text_field.dart:87]: I see the same correction in another PR 🤔 
https://github.com/trashfeed/input_history_text_field/pull/9https://github.com/trashfeed/input_history_text_field/pull/9

**Review by trashfeed — COMMENTED** [review]
@fuku02 
Sorry for the late review.
Thank you for your excellent work！

There are some changes that are duplicates of other PRs and the differences seem to be getting bigger. Please only make the corrections that are really needed as I am unable to review 🙏.

**Comment by trashfeed** [comment]
@fuku02 
Thanks for ↓ PR. 
https://github.com/trashfeed/input_history_text_field/pull/9
We made some modifications to release it. Sorry. This caused a conflict.🙏

**Review by cristiandonaraDEV — APPROVED** [review]
Everything is fine, also the overlay removes fixes a lot of bugs

**Comment by cristiandonaraDEV** [comment]
please merge this. works fine

**Pull request closed without merging**